src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -242,7 +242,7 @@
     <comment>Description for Formula</comment>
   </data>
   <data name="FormulaPortDataResultToolTip" xml:space="preserve">
-    <value>Result of math computation</value>
+    <value>String representation of the object.</value>
```

Comment:
Is this right?

Reply:
![image](https://user-images.githubusercontent.com/25138291/92095493-a88b4700-edcd-11ea-9b3a-d6de853c3e29.png)
This is the existing node, I think the description of the output makes no sense

Reply:
Hahaha, yep your description makes way more sense!

Reply:
i see, still dont think this node should use `FormulaPortDataResultToolTip` as changing this will also change the `Formula` nodes output tooltip. If you look at the Resource file there is a `FromObjectPortDataResultToolTip` already use that for the `String from Object` node

Reply:
Resolved by using appropiate Tooltip